### PR TITLE
Added Logout Route

### PR DESCRIPTION
**Pull request by mark-nakachon**
### Checklist:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you successfully ran tests with your changes locally?
* [x] Have you added an explanation of what your changes do and why you'd like us to include them?

### Issues Solved

Fixes #{98}

### CODE REVIEW

**Comment by mark-nakachon** [comment]
Please review my PR

File: server/index.js [hunk 1]
```diff
@@ -166,6 +167,10 @@ app.post('/login', (req, res, done) => {
 		successRedirect: '/'
 	})(req, res, done);
 });
+app.get('/logout', authenticated, (req, res) => {
```
Comment (rossoskull) [inline server/index.js]: You should use `post` method instead of `get`, as browsers will prefetch these routes, and thus log the user out automatically

**Comment by mark-nakachon** [comment]
Changed to Post from get

**Comment by mark-nakachon** [comment]
Resolved conflicts

**Pull request merged**